kubernetes/celestia-local/scripts/init-celestia-appd.sh

```diff
@@ -23,7 +26,11 @@ celestia-appd gentx \
   --keyring-backend="$keyring_backend" \
   --chain-id "$chainid" \
   --home "$home_dir" \
-  --orchestrator-address "$validator_key" \
   --evm-address "$evm_address"
 
 celestia-appd collect-gentxs --home "$home_dir"
+
+# update timeout_propose in config.toml
```

Comment:
nit: empty line below, can we make this comment something like "speed up celestia block times"